charts/seed-monitoring/charts/alertmanager/templates/allow-alertmanager-network-policy.yaml

```diff
@@ -21,5 +21,12 @@ spec:
       namespaceSelector:
```

Comment:
I now activate only the relevant network policy

Reply:
To support migration cases we would need to change this in the code where we also migrate the [ingressClass](https://github.com/gardener/gardener/pull/3131/files#diff-8495b57b25d39744fe1b20b1e1ce695d130f730142af86ff9c5929afcdbabeb4R798). This seems like complicating the code without a real security benefit. We (@rfranzke) think this could be done in a separate refactoring of network policies where the netpol allowing traffic to/from prometheus should be created in the component (like nginx-ingress) instead of all network policies being defined in [prometheus](https://github.com/gardener/gardener/blob/master/charts/seed-monitoring/charts/core/charts/prometheus/templates/allow-prometheus-network-policy.yaml).